cmd/kubelet/app/options/options.go

```diff
@@ -80,6 +80,10 @@ type KubeletFlags struct {
 
 	// Container-runtime-specific options.
 	ContainerRuntimeOptions
+
+	// Experimental flag that enables shared/slave propagation for mounts in
+	// pods. It may be redesigned or even removed in future releases!
+	MountPropagation bool
```

Comment:
We talked about this specific flag in sig-node and agreed it was ok for now. Can you please name it `ExperimentalMountPropagation` though (and the flag `--experimental-mount-propagation`), just to make it really clear that it's experimental?

We don't want to keep adding experimental stuff to the Kubelet's versioned configuration types at the moment, so the only option is to add flags for experimental things. At some point (hopefully soon) apimachinery will have field gates (https://docs.google.com/document/d/1wuoSqHkeT51mQQ7dIFhUKrdi3-1wbKrNWeIL4cKb9zU/edit#), and we can probably use that to include experimental things in the versioned configuration type (without having to rev the version), rather than as flags.